js/src/forum/compat.js

```diff
@@ -72,6 +71,7 @@ import Search from './components/Search';
 import DiscussionListItem from './components/DiscussionListItem';
 import LoadingPost from './components/LoadingPost';
 import PostsUserPage from './components/PostsUserPage';
+import EditUserModal from '../common/components/EditUserModal';
```

Comment:
This should be labelled as deprecated

Reply:
I've moved this to the common `compat.js`. This should be combined with the other `compat`s and mean that people importing `flarum/components/EditUserModal` should be fine, right?

Will having this be exported from common and forum's `compat.js` cause issues?

Reply:
If someone is importing via the new import scheme, this is a breaking change.

Reply:
The new scheme that wasn't announced? 😝 

You make a good point, I'll re-add it!

Reply:
Public relations fails aside, public API is public API :shrug: :hear_no_evil: